u256.go

```diff
@@ -68,10 +67,13 @@ func (u u256) cmp128(v u128) int {
 	return u128FromHiLo(u.hi, u.lo).Cmp(v)
 }
 
+// pow returns u^e (with e > 0)
+// Use int instead of uint to avoid unnecessary checks for type conversion
 func (u u256) pow(e int) (u256, error) {
-	if e <= 0 {
-		return u256{}, fmt.Errorf("invalid exponent %d. Must be greater than 0", e)
-	}
+	// e should be greater than 0 already. No need to check
+	// if e <= 0 {
+	// 	return u256{}, fmt.Errorf("invalid exponent %d. Must be greater than 0", e)
+	// }
```

Comment:
You left commented code

Reply:
Thanks. Missing this one

Reply:
I would recomment you

https://go-critic.com/overview.html#commentedoutcode